src/contracts/core/AllocationManager.sol

```diff
@@ -403,6 +438,27 @@ contract AllocationManager is
         return (freeMagnitudeToAdd, nextIndex);
     }
 
+    /**
+     * @notice Returns the allocation delay of an operator
+     * @param operator The operator to get the allocation delay for
+     * @dev Defaults to `DEFAULT_ALLOCATION_DELAY` if none is set
+     */
+    function allocationDelay(
+        address operator
+    ) public view returns (uint32 delay) {
```

Comment:
perhaps return a boolean as well indicating whether or not the operator has ever set an allocationDelay.
This could be checked by `pendingDelayEffectTimestamp != 0`
If it returns false, modifyAllocations should revert